src/composables/accounts.ts

```diff
@@ -300,21 +299,16 @@ export function useAccounts() {
     }
   }
 
-  async function openSetPasswordModal(newMnemonic: string, isRestored = false) {
-    const { openModal } = useModals();
-
-    const password = await openModal<string>(MODAL_SET_PASSWORD, {
-      isRestoredWallet: isRestored,
-    });
-
-    await setPasswordAndEncryptMnemonic(newMnemonic, password);
```

Comment:
This didn't make sense to me that the `openSetPasswordModal` modal was also encrypting the mnemonic. I removed this line and used it in the `accounts` composable.

The method was moved to `modals` composable.